public/app/features/alerting/unified/components/contact-points/useContactPoints.tsx

```diff
@@ -296,13 +296,14 @@ export function useContactPointsWithStatus({
     ...defaultOptions,
     selectFromResult: (result) => ({
       ...result,
-      contactPoints: result.data
-        ? enhanceContactPointsWithMetadata({
-            notifiers: cloudNotifierTypes,
-            contactPoints: result.data.alertmanager_config.receivers ?? [],
-            alertmanagerConfiguration: result.data,
-          })
-        : [],
+      contactPoints:
+        result.data && !isGrafanaAlertmanager // I needed to add this condition to avoid the error in tests, it seems that the skip is not working as expected
```

Comment:
without this doble check (`!isGrafanaAlertManager` in here and in the skip option), tests were not skipping, and the result used was from the alert manager configuration....